app/src/main/java/net/squanchy/settings/SettingsFragment.java

```diff
@@ -89,6 +105,74 @@ public void onCreate(Bundle savedInstanceState) {
         });
     }
 
+    private void onGoogleConnectionFailed() {
+        Timber.e("Google Client connection failed");
+        Snackbar.make(getViewOrThrow(), R.string.proximity_error_google_client_connection, Snackbar.LENGTH_LONG).show();
+    }
+
+    private ProximityPreconditions.Callback proximityPreconditionsCallback() {
+        return new ProximityPreconditions.Callback() {
+            @Override
+            public void notOptedIn() {
+                showProximityEnablingError(Snackbar.make(getViewOrThrow(), R.string.proximity_error_not_opted_in, Snackbar.LENGTH_LONG));
```

Comment:
Done already in 615db96ffbea849e8c230ee381f2197880954e84 addressing the equivalent comment in `LocationOnboardingActivity`. This was once used to offer actions in snackbars, then I took them out and forgot to re-simplify this. Sorry.